crates/infra/cli/src/toolchains/wasm/mod.rs

```diff
@@ -92,10 +91,25 @@ impl WasmPackage {
                 .property("--out-dir", temp_dir.unwrap_str())
                 .property("--base64-cutoff", "0") // disable inlining core Wasm binaries as base64
                 .flag("--no-namespaced-exports") // disable namespaced exports for typescript compatibility
+                .flag("--no-typescript") // disable generating .d.ts files - we'll do this in `generate_types` with a better generator
```

Comment:
nit: outdated comment?

```suggestion
                .flag("--no-typescript") // disable generating .d.ts files - we'll do this in `jco types` below
```